Modules/Core/Common/include/itkProgressAccumulator.h

```diff
@@ -138,6 +138,8 @@ class ITKCommon_EXPORT ProgressAccumulator : public Object
     // The tags for adding/removing observers to mini-pipeline filter
     unsigned long ProgressObserverTag;
     unsigned long StartObserverTag;
+
+    bool AccumulatedProgress{ 0.0 };
```

Comment:
I am getting an error here with conversion from double to bool.

Reply:
I changed the variable name and how it is used in code, but forgot to change the type. And I was wondering why does debugger keep representing it as a float. Silly me! Thank you Brad.